### PR TITLE
Add support for NHI GWO database

### PR DESCRIPTION
**Pull request by rubencalje**
This pull request adds support for the NHI GWO databse with extraction data in the Netherlands, mainly form drinking water companies:
https://gwo.nhi.nu/

Users should register at this website to obtain a username and password. For testing purposes, I added my username and password to the Github-secrets, which are used by the tests.

### CODE REVIEW

File: tests/test_021_nhi.py [hunk 1]
```diff
@@ -20,3 +22,44 @@ def test_buidrainage():
...
+
+    # download extractions from well "13-PP016" of pomping station Veghel
+    measurements, gdf = nlmod.read.nhi.get_gwo_measurements(
+        username, password, well_site="veghel", filter__well__name="13-PP016"
+    )
+    assert measurements.reset_index()["Name"].isin(gdf.index).all()
+
+
+@pytest.mark.skip("too slow")
+def test_gwo_entire_pumping_station():
```
Comment (dbrakenhoff) [inline tests/test_021_nhi.py:45]: out of curiosity, how long does this request take?

**Review by dbrakenhoff — COMMENTED** [review]
Looks good, I made some typo corrections, and just the one question about the hard-coded index. If that's cleared up, it's good to go!

File: nlmod/read/nhi.py [hunk 2]
```diff
@@ -173,3 +177,189 @@ def add_buisdrainage(
...
+        params.update(kwargs)
+        r = requests.get(url, auth=(username, password), params=params, timeout=timeout)
+
+        content = r.content.decode("utf-8")
+        lines = content.split("\n")
+        empty_lines = np.where([set(line) == set(";") for line in lines[:100]])[0]
+        assert len(empty_lines) == 2
+
+        # read properties
+        skiprows = list(range(empty_lines[0] + 1)) + [empty_lines[0] + 2]
```
[the remark this answers is not in the document]
Comment (dbrakenhoff) [reply nlmod/read/nhi.py:335]: Comment got moved I see, the line in question is a few lines before.

[the remark this answers is not in the document]
Comment (rubencalje) [reply nlmod/read/nhi.py:335]: If there are less than 100 lines in the returned content, this code still works.

However, there are some situations, where no wells or measurements are found, that will give an unclear error, that is not caught yet. I will try to fix this.

[the remark this answers is not in the document]
Comment (rubencalje) [reply nlmod/read/nhi.py:355]: But there may be more than 100 wells, then this will fail... I removed the 100

**Pull request merged**